tools/project_manager/editproject.php

```diff
@@ -184,31 +184,20 @@ function set_from_marc_record()
     {
         global $pguser;
 
-        if (!isset($_POST['rec']))
+        $encoded_marc_array = array_get($_POST, "rec", "");
+        if(!$encoded_marc_array)
         {
-            return sprintf(_("parameter '%s' is unset"), 'rec');
+            return sprintf(_("No record selected. If no results are suitable, select '%s' to create the project manually."), "No Matches");
```

Comment:
"No Matches" should part of the translated string (because the referenced button's text is itself translated).

Reply:
I did it this way on purpose so that the "No Match" string pulls from the same translation to prevent the two strings from accidentally being translated differently, except that I forgot to wrap the string in `_()` -- I'll fix that.

Reply:
Okay. In that case, external_catalog_search.php (at about line 141) has another string referring to that button that you might want to treat similarly.